hash/snes.xml

```diff
@@ -3541,6 +3566,30 @@ Beyond that last category are the roms waiting to be classified.
 		</part>
 	</software>
 
+	<software name="superman">
+		<!-- snescentral.com -->
+		<description>Superman Unfinished Version (demo)</description>
```

Comment:
Judging by the title screen from https://youtube.com/watch?v=jRsEBvgA33E, "Unfinished Version" is a subtitle, so it should read "Superman - Unfinished Version"